### ISSUE TITLE
Feature-Request: "Working Week View"

### ISSUE THREAD

**Issue by dewdeded** [valincius/BlazorScheduler#3]
Feature-Request: Implement a "working week view"

- Package would be usefuller, if it would be possible to configure have an view/option to just show the work week days from Monday till Friday

**Comment by valincius** [comment]
![image](https://user-images.githubusercontent.com/15176357/125174332-901c5380-e179-11eb-8846-3b5c77704784.png)

Is this what you're looking for? (this is taken from Google calendar)

**Comment by dewdeded** [comment]
I need something like this: 
**Only Monday till Friday (no weekend)**
https://www.kribbelbunt.de/fileadmin/user_upload/stundenplan-vorlage.png

**Or: Monday till Friday as reguilar days and Saturday/Sunday reduced/smaller**
https://www.kalenderpedia.de/images-large/wochenkalender/wochenkalender--2020.png

Thanks.

**Comment by valincius** [comment]
So the first option (hiding weekend) won't work so well with the dragging & it'll be kind of weird for appointments that span multiple weeks.

Would making weekends more narrow work for you? Like so:
![image](https://user-images.githubusercontent.com/15176357/125175472-48012f00-e181-11eb-8a1f-8636c80ab261.png)

**Comment by SeanFeldman** [comment]
@valincius is it possible to render a weekly view similar to Google, as you've shown [here](https://github.com/valincius/BlazorScheduler/issues/3#issuecomment-877691039)?

**Comment by valincius** [comment]
@SeanFeldman 
This is not an option currently, but feel free to create an issue and I will try to get started on it as soon as I can

**Comment by SeanFeldman** [comment]
Would it be possible to re-open this issue for tracking that work? This one is exactly what I was looking and that's how I found it rather than creating a duplicate 🙂

**Comment by valincius** [comment]
@SeanFeldman you're right, I thought this was for something different. Yes I will do that. Thank you!

**Comment by bluisanajmr** [comment]
I am looking for a scheduling component for a SaaS app we are building.  This is the only feature that is stopping me from using this component in our app.

**Comment by valincius** [comment]
@bluisanajmr hi, thanks for considering using this project. I am planning to completely rewrite this library soon - I hadn't originally considered alternate views when I was building this so it's going to take a lot of hacking together if I were to try to add that in but I will be sure to include this feature in the rewrite - I cannot give you a timeline on when that may come though.

--edit

I've actually started the rewrite of this today. It's going fairly quickly but I still can't guarantee when it'll be done.

**Comment by valincius** [comment]
Sorry it took so long. Fixed in v5.0.1
